src/core/Akka.Streams.Tests/Dsl/FlowLogSpec.cs

```diff
@@ -26,145 +28,169 @@ public FlowLogSpec(ITestOutputHelper helper) : base("akka.loglevel = DEBUG", hel
         {
             var settings = ActorMaterializerSettings.Create(Sys).WithInputBuffer(2, 16);
             Materializer = ActorMaterializer.Create(Sys, settings);
+        }
+
+        private TestProbe _logProbe;
 
+        protected override void AtStartup()
+        {
+            base.AtStartup();
+            
             var p = CreateTestProbe();
-            Sys.EventStream.Subscribe(p.Ref, typeof(object));
-            LogProbe = p;
+            Sys.EventStream.Subscribe(p.Ref, typeof(LogEvent));
```

Comment:
Don't subscribe to everything, we're only interested in `Debug`, `Info`, `Warning` and `Error`